packages/server/lib/server.ts

```diff
@@ -38,6 +38,7 @@ import { getLogger } from '@nangohq/utils/dist/logger.js';
 import { getGlobalOAuthCallbackUrl, environmentService, getPort, errorManager, getWebsocketsPath, packageJsonFile } from '@nangohq/shared';
 import oAuthSessionService from './services/oauth-session.service.js';
 import migrate from './utils/migrate.js';
+import { migrate as migrateRecords } from '@nangohq/records/dist/db/migrate.js';
```

Comment:
Can we have a barrel file instead? same issue as `utils` typescript can't autosuggest this file in vscode

Reply:
also I would not ship this PR with this, so you can merge right now and try later

Reply:
good point about not shipping

Reply:
also added a barrel file